Source/Core/InputCommon/ControllerEmu/ControllerEmu.h

```diff
@@ -14,7 +14,11 @@
 
 class ControllerInterface;
 
-#define sign(x) ((x) ? (x) < 0 ? -1 : 1 : 0)
+template <typename T>
```

Comment:
While a good change, this doesn't seem to be related to using Eigen as far as I can tell and should probably go in its own commit.

Reply:
This change is needed to fix compile errors where "sign" is used as an identifier.